### ISSUE TITLE
Install and Set Up kubectl on Windows - del command update

### ISSUE THREAD

**Issue by Kheiden** [kubernetes/website#45468]
When following the instructions, I received the following error message when trying to run this command:

`del kubectl-convert.exe kubectl-convert.exe.sha256` 

![2024-03-07 12_31_54-Administrator_ Windows PowerShell](https://github.com/kubernetes/website/assets/8250577/f576b360-1924-48d7-bc0b-bb0cb4001db8)

Text:
```
Remove-Item : A positional parameter cannot be found that accepts argument 'kubectl-convert.exe.sha256'.
At line:1 char:1
+ del kubectl-convert.exe kubectl-convert.exe.sha256
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Remove-Item], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.RemoveItemCommand
```

However running the following commands worked:

```
del kubectl-convert.exe
del kubectl-convert.exe.sha256
```

I recommend updating the page.

**Comment by dipesh-rawat** [comment]
Page mentioned in the issue _(based on the current title)_: https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/
/language en

**Comment by dipesh-rawat** [comment]
Looks like a bug to me.
/kind bug

**Comment by Kheiden** [comment]
I've created a PR to address this minor documentation fix:

https://github.com/kubernetes/website/pull/45469

**Comment by vaibhav2107** [comment]
I think this issue has been resolved by this PR #45469 and hence we can close this issue
